application/routes.js

```diff
@@ -378,6 +378,101 @@ module.exports = function(server) {
     },
   });
 
+  server.route({
+    method: 'POST',
+    path: '/PRvideo',
+    handler: handlers.createPRVideo,
+    config: {
+    //   auth: 'jwt',
+      cors: {
+        origin: ['*'],
+        additionalHeaders: ['----jwt----']
+      },
+      payload: {
+        output: 'file',
+        uploads: '/tmp/',
+        maxBytes: 10485760, //100MB
+        parse: true,
+        allow: 'multipart/form-data',
+        failAction: 'log'
+      },
+      validate: {
+        options: { convert: true },
+        payload: Joi.object({
+          audioFile: Joi.object().required(),
+          slideTimings: Joi.string().trim().min(5).required()
```

Comment:
Please add as comment or in the route description what are the requirements (like why slideTimings needs to be at least 5 pparts)